IBAnimatable/MaskDesignable.swift

```diff
@@ -141,35 +110,27 @@ public extension MaskDesignable where Self: UIView {
   }
   
   // MARK: - Parallelogram
-    
-  private func maskParallelogramFromString(_ mask: String) {
-    if let angle = Double(retrieveMaskParameters(mask, maskName: MaskType.parallelogram.rawValue)) {
-      maskParallelogram(angle)
-    } else {
-      maskParallelogram()
-    }
-  }
   
-  private func maskParallelogram(_ topLeftAngle: Double = 60) {
+  private func maskParallelogram(_ topLeftAngle: Double) {
     let parallelogramPath = maskParallelogramBezierPath(topLeftAngle)
     drawPath(parallelogramPath)
   }
   
   private func maskParallelogramBezierPath(_ topLeftAngle: Double) -> UIBezierPath {
-    let topLeftAngleRad = Double(topLeftAngle) * M_PI / 180
+    let topLeftAngleRad = topLeftAngle * .pi / 180
     let path = UIBezierPath()
-    let offset = abs(CGFloat(tan(topLeftAngleRad - M_PI / 2)) * bounds.height)
+    let offset = abs(CGFloat(tan(topLeftAngleRad - .pi / 2)) * bounds.height)
     
     if topLeftAngle <= 90 {
-      path.move(to: CGPoint(x: 0, y: 0))
+      path.move(to: CGPoint(x:0, y:0))
```

Comment:
There should be an issue normal, SwiftLint syntax is `x: 0, y: 0` instead of `x:0,y:0`. It would be worth it to find out why there's no warnings about this